### PR TITLE
text-transform 

### PR DESCRIPTION
**Pull request by enchev**
Implemented #685

### CODE REVIEW

File: ui/styling/style.ts [hunk 1]
```diff
@@ -566,6 +577,12 @@ export class Style extends DependencyObservable implements styling.Style {
         }
     }
 
+    public _updateTextTransform() {
+        if (this._getValue(textTransformProperty) !== enums.TextTransform.none) {
```
Comment (hshristov) [inline ui/styling/style.ts]: Will it works if you set textTransform to 'uppercase' and then to 'none'?

Comment (enchev) [inline ui/styling/style.ts]: This code is not related to changing textTransform property but to perform some other stuff if the  textTransform is set (not none).

**Comment by hamorphis** [comment]
:+1:

**Comment by nsndeck** [comment]
:+1:

**Pull request merged**